components/LifeAtTorchbox/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from './MainContent.module.scss';
+
+type MainContentProps = {
+    firstLine: string;
+    secondLine: string;
+    thirdLine: string;
+    children: React.ReactNode;
+};
+
+export const MainContent = ({
+    firstLine,
+    secondLine,
+    thirdLine,
+    children,
+}: MainContentProps) => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.textContainer}>
+                <div className={styles.titleContainer}>
+                    <h2 className={styles.title}>
+                        {firstLine}
+                        <br aria-hidden="true" />
```

Comment:
This aria-hidden stops the <br> being read out as a pause for screenreader users. Note that we're using fullstops for each word in the existing designs, so screenreader users should still hear a pause anyway.